modules/optimization/include/executioners/OptimizeSolve.h

```diff
@@ -149,6 +149,9 @@ class OptimizeSolve : public SolveObject
   /// Parameters (solution)
   std::unique_ptr<libMesh::PetscVector<Number>> _parameters;
 
+  /// Iteration number for the forward problem
```

Comment:
This is really for the forward problem/solve?

Reply:
It's for the optimization process; for steady and adjoint should be forward-adjoint solve. But Lynn wanted to extend it to plain `Steady` too. So I switched the comment to "Optimization iteration number".